src/lib/helpers/config.ts

```diff
@@ -19,6 +19,7 @@ const ConfigSchema = z.object({
     }).strict().default({}),
     networking: z.object({
         ump: z.boolean().default(false),
+        video_fetch_chunk_size_mb: z.number().default(5),
```

Comment:
is this a new parameter? could you add it to the config.example.toml?

Reply:
oh, good catch.